modules/web/js/ballerina/utils/ace-styles.css

```diff
@@ -10,3 +10,63 @@
 .text-editor.ace_editor {
     font: 12px/normal 'Droid Sans Mono', monospace;
 }
+
```

Comment:
Shall we add a comment specifically mentioning what the use case we are addressing here